db/warehouse/migrate/20250313125655_add_relevant_state_to_site_config.rb

```diff
@@ -0,0 +1,8 @@
+
+# frozen_string_literal: true
+
+class AddRelevantStateToSiteConfig < ActiveRecord::Migration[7.0]
+  def change
+    add_column :configs, :relevant_state_codes, :string, default: ENV['RELEVANT_COC_STATE'] || 'MA', null: false
```

Comment:
Defaulting to MA for new installations feels a little weird, I thought we are trying to move away from MA-defaulted text and code? Is the issue that everything will break if there's no state?

Reply:
That's exactly the issue.  There's an assumption that it is set.  It shouldn't actually default to MA unless we don't have `ENV['RELEVANT_COC_STATE']`, which should be set correctly prior to DB setup... Except, MA probably ends up in the structure file.  Yeah, not ideal, but things start to break if there's no state.  At least it's easily fixable now.